Notification.js

```diff
@@ -65,17 +65,25 @@ class Notification extends Component {
   }
 
   showNotification(done = () => {}) {
+    this.props.onShowing && this.props.onShowing();
```

Comment:
Can you use default props here instead of using a `&&` gate. That way we can always call `this.props.onShowing`, and we know what its type is.

Reply:
done